packages/main/src/components/DynamicPage/DynamicPage.jss.ts

```diff
@@ -45,7 +45,7 @@ export const styles = {
   },
   contentContainer: {
     extend: ResponsiveContainerPadding,
-    paddingTop: '1rem',
+    paddingBlockEnd: '1rem',
```

Comment:
Is this change intended? `paddingTop` should translate to `paddingBlockStart`, shouldn't it?

Reply:
Nope, that wasn't intended. Good catch!